design-editor/src/pane/design-editor-element.js

```diff
@@ -915,6 +890,52 @@ class DesignEditor extends DressElement {
 		}
 	}
 
+	/**
+	 * Check if dropped widget has a container
+	 * @param {Object} element
+	 */
+
+	checkisElementHasContainer(element) {
```

Comment:
I'm not sure about function name. This method return container so function name should be like "getConteinerByElement" or similar. Methods prefixed by (is) like "isDone" should return true / false.